var/spack/repos/builtin/packages/fraggenescan/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fraggenescan(MakefilePackage):
+    """FragGeneScan is an application for finding (fragmented) genes in short
+       reads. It can also be applied to predict prokaryotic genes in
+       incomplete assemblies or complete genomes."""
+
+    homepage = "https://sourceforge.net/projects/fraggenescan/"
+    url      = "https://downloads.sourceforge.net/project/fraggenescan/FragGeneScan1.31.tar.gz"
+
+    version('1.31', sha256='cd3212d0f148218eb3b17d24fcd1fc897fb9fee9b2c902682edde29f895f426c')
+
+    build_targets = ['fgs']
+
+    def edit(self, spec, prefix):
+        filter_file('gcc', self.compiler.cc, 'Makefile', string=True)
```

Comment:
```suggestion
        filter_file('gcc', spack_cc, 'Makefile', string=True)
```
`self.compiler.cc` is the actual compiler, we want to use Spack's compiler wrapper instead

Reply:
My mistake -- I assumed they pointed to the same value.